src/components/theme/Icon/Icon.jsx

```diff
@@ -32,12 +32,13 @@ const defaultSize = '36px';
  *
  * for further reference see {@link https://kitconcept.com/blog/pastanaga-icon-system/ | here}
  */
-const Icon = ({ name, size, color, className, title, onClick }) => (
+const Icon = ({ name, size, color, className, title, onClick, tabIndex }) => (
   <svg
     xmlns={name.attributes && name.attributes.xmlns}
     viewBox={name.attributes && name.attributes.viewBox}
     style={{ height: size, width: 'auto', fill: color || 'currentColor' }}
     className={className ? `icon ${className}` : 'icon'}
+    tabIndex={tabIndex}
```

Comment:
@iFlameing I think we should make tabIndex default to 0 if it is undefined AND onClick is not undefined. That way icons with a click handler will always be reachable by the keyboard, even if the developer using the Icon element does not remember to add the tabindex. Does that make sense?